app/models/manageiq/providers/ibm_cloud/vpc/cloud_manager/provision_workflow/network.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+require 'ipaddr'
```

Comment:
Suggest moving this require closer to where it is used around line 64.

Reply:
See #203